samples/HostFilteringSample/Startup.cs

```diff
@@ -23,9 +26,22 @@ public void ConfigureServices(IServiceCollection services)
         {
             services.AddHostFiltering(options =>
             {
-                // If this is excluded then it will fall back to the server's addresses
-                options.AllowedHosts = Config.GetSection("AllowedHosts").Get<List<string>>();
+
```

Comment:
Is there a reason you prefer doing this in PostConfigure instead of in the Middleware, perhaps in the EnsureConfig method that computes the list instead?

That is generally safer than assuming no-one adds a PostConfigure after you

Reply:
It's a compromise between Damian and David to avoid direct config lookups in the middleware. CreateDefaultBuilder has config layout opinions, not the individual components.